sorc/ncep_post.fd/CALDRG.f

```diff
@@ -1,42 +1,20 @@
 !> @file
-!
-!> SUBPROGRAM:    CALDRG      COMPUTE DRAG COEFFICIENT
-!!   PRGRMMR: TREADON         ORG: W/NP2      DATE: 93-09-01
-!!     
-!! ABSTRACT:  THIS ROUTINE COMPUTES A SURFACE LAYER DRAG
-!!   COEFFICIENT USING EQUATION (7.4.1A) IN "AN INTRODUCTION
-!!   TO BOUNDARY LAYER METEOROLOGY" BY STULL (1988, KLUWER
-!!   ACADEMIC PUBLISHERS).
-!!     
-!! PROGRAM HISTORY LOG:
-!!   93-09-01  RUSS TREADON
-!!   98-06-15  T BLACK - CONVERSION FROM 1-D TO 2-D
-!!   00-01-04  JIM TUCCILLO - MPI VERSION           
-!!   02-01-15  MIKE BALDWIN - WRF VERSION
-!!   05-02-22 H CHUANG - ADD WRF NMM COMPONENTS 
-!!     
-!! USAGE:    CALL CALDRG(DRAGCO)
-!!   INPUT ARGUMENT LIST:
-!!     NONE     
-!!
-!!   OUTPUT ARGUMENT LIST: 
-!!     DRAGCO   - SURFACE LAYER DRAG COEFFICIENT
-!!     
-!!   OUTPUT FILES:
-!!     NONE
-!!     
-!!   SUBPROGRAMS CALLED:
-!!     UTILITIES:
-!!       NONE
-!!     LIBRARY:
-!!       COMMON   - LOOPS
-!!                  SRFDSP
-!!                  PVRBLS
-!!     
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN 90
-!!     MACHINE : CRAY C-90
-!!
+!> @brief Subroutine that computes drag cofficient.
```

Comment:
On the NCEPLIBS documentation, we put an author for every file, and for every subroutine in the file.